common/features.c

```diff
@@ -203,9 +153,11 @@ static void clear_feature_bit(u8 *features, u32 bit)
  *  - MUST set `len` to the minimum length required to hold the `features` bits
  *  it sets.
  */
-u8 *get_agreed_channelfeatures(const tal_t *ctx, const u8 *theirfeatures)
+u8 *get_agreed_channelfeatures(const tal_t *ctx,
+			       const struct feature_set *ours,
+			       const u8 *theirfeatures)
```

Comment:
Nit: inconsistent naming, should be `theirs` and `ours, or `theirfeatures` and `ourfeatures`.